### PR TITLE
Translations for Ethers.rs, Tenderly, and Ledger Live

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

- Ledger Live: https://github.com/PureStake/moonbeam-docs/pull/485 & https://github.com/PureStake/moonbeam-docs/pull/492
- Tenderly: https://github.com/PureStake/moonbeam-docs/pull/490
- Ethers.rs: https://github.com/PureStake/moonbeam-docs/pull/484

### Checklist

- [ ] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [ ] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [ ] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

n/a

### CODE REVIEW

File: builders/build/eth-api/dev-env/tenderly.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+- **[Gas Profiler](https://docs.tenderly.co/debugger/how-to-use-tenderly-debugger#gas-profiler){target=_blank}** - 查看您更细化的gas花费情况，从而您可以优化您的智能合约并减少交易gas成本
+
+- **[Simulator](https://docs.tenderly.co/simulations-and-forks/how-to-simulate-a-transaction){target=_blank}** - 在分叉的开发环境中模拟交易，以在无需将交易传送至链上的情况下了解其如何运作。通过这种方式，您可以了解交易的结果并确保在将其发送至网络前按预期运作。您可以尝试不同的参数，模拟历史和当前交易，并编辑合约源代码。您可以从Tenderly数据面板，或在[Tenderly Simulation API](https://docs.tenderly.co/simulations-and-forks/simulation-api){target=_blank}以编程的方式利用模拟器来访问模拟器
+
+- **[Forks](https://docs.tenderly.co/simulations-and-forks/how-to-create-a-fork){target=_blank}** - 此功能在隔离环境中模拟实时Moonbeam网络，这使您可以与部署的合约和实时链上数据进行交互。您能够按时间顺序将多个模拟链接在一起，分叉还将使交易模拟更进一步。Forks允许复杂的交易测试场景，当出现一个交易依赖于另一个交易，并具有使用实时链上数据的好处。使用Tenderly分叉功能的时候需要注意一些限制。您不能与任何Moonbeam预编译的合约及其功能交互。预编译是Substrate实施的一部分，因此不能在模拟的EVM环境中复制。这禁止您在Moonbeam和基于Substrate的功能（如质押和治理）上与跨链资产交互
+
+- **[Alerting](https://docs.tenderly.co/alerts/alerting){target=_blank}** - 配置实时警报，在特定活动发生时通知，以便让您了解智能合约的情况
```
Comment (eshaben) [inline builders/build/eth-api/dev-env/tenderly.md:26]:
```suggestion
- **[Alerting](https://docs.tenderly.co/alerts/intro-to-alerts){target=_blank}** - 配置实时警报，在特定活动发生时通知，以便让您了解智能合约的情况
```

Fixing this link on the EN site, so wanted to make sure it doesn't get missed here

File: .snippets/text/ledger/checking-prereqs.md [hunk 2]
```diff
@@ -1,14 +1,9 @@
...
- - [Ledger Nano S firmware v2.0.0](https://support.ledger.com/hc/en-us/articles/360010446000-Ledger-Nano-S-firmware-release-notes?docs=true)
- - [Ledger Nano X firmware v2.0.1](https://support.ledger.com/hc/en-us/articles/360014980580-Ledger-Nano-X-firmware-release-notes?docs=true)
-
-另外，您需要MetaMask作为Ledger设备和Moonbeam网络之间的媒介。确保您的MetaMask[已连接至Moonbeam](/tokens/connect/metamask/)。
-
-自[MetaMask 10.5.0版本](https://consensys.net/blog/metamask/metamask-and-ledger-integration-fixed/)起，在Google Chrome上将您的Ledger设备连接至MetaMask颇为方便，您只需安装最新版本的MetaMask即可。
-
+ - [Ledger Live 2.35.1](https://support.ledger.com/hc/en-us/articles/360020773319-What-s-new-in-Ledger-Live-?docs=true){target=_blank}
+ - [Ledger Nano S firmware v2.0.0](https://support.ledger.com/hc/en-us/articles/360010446000-Ledger-Nano-S-firmware-release-notes?docs=true){target=_blank}
+ - [Ledger Nano X firmware v2.0.1](https://support.ledger.com/hc/en-us/articles/360014980580-Ledger-Nano-X-firmware-release-notes?docs=true){target=_blank}
```
Comment (themacexpert) [inline .snippets/text/ledger/checking-prereqs.md:9]:
```suggestion
 - [Ledger Nano X firmware v2.0.1](https://support.ledger.com/hc/zh-cn/articles/360014980580-Ledger-Nano-X-%E5%9B%BA%E4%BB%B6%E7%89%88%E6%9C%AC%E8%AF%B4%E6%98%8E?docs=true){target=_blank}
```

File: .snippets/text/ledger/checking-prereqs.md [hunk 3]
```diff
@@ -1,14 +1,9 @@
...
- - [Ledger Live 2.35.1](https://support.ledger.com/hc/en-us/articles/360020773319-What-s-new-in-Ledger-Live-?docs=true)
- - [Ledger Nano S firmware v2.0.0](https://support.ledger.com/hc/en-us/articles/360010446000-Ledger-Nano-S-firmware-release-notes?docs=true)
- - [Ledger Nano X firmware v2.0.1](https://support.ledger.com/hc/en-us/articles/360014980580-Ledger-Nano-X-firmware-release-notes?docs=true)
-
-另外，您需要MetaMask作为Ledger设备和Moonbeam网络之间的媒介。确保您的MetaMask[已连接至Moonbeam](/tokens/connect/metamask/)。
-
-自[MetaMask 10.5.0版本](https://consensys.net/blog/metamask/metamask-and-ledger-integration-fixed/)起，在Google Chrome上将您的Ledger设备连接至MetaMask颇为方便，您只需安装最新版本的MetaMask即可。
-
+ - [Ledger Live 2.35.1](https://support.ledger.com/hc/en-us/articles/360020773319-What-s-new-in-Ledger-Live-?docs=true){target=_blank}
+ - [Ledger Nano S firmware v2.0.0](https://support.ledger.com/hc/en-us/articles/360010446000-Ledger-Nano-S-firmware-release-notes?docs=true){target=_blank}
```
Comment (themacexpert) [inline .snippets/text/ledger/checking-prereqs.md:8]:
```suggestion
 - [Ledger Nano S firmware v2.0.0](https://support.ledger.com/hc/zh-cn/articles/360010446000-Ledger-Nano-S-%E5%9B%BA%E4%BB%B6%E7%89%88%E6%9C%AC%E8%AF%B4%E6%98%8E?docs=true){target=_blank}
```

File: .snippets/text/ledger/checking-prereqs.md [hunk 4]
```diff
@@ -1,14 +1,9 @@
...
 
- - [Ledger Live 2.35.1](https://support.ledger.com/hc/en-us/articles/360020773319-What-s-new-in-Ledger-Live-?docs=true)
- - [Ledger Nano S firmware v2.0.0](https://support.ledger.com/hc/en-us/articles/360010446000-Ledger-Nano-S-firmware-release-notes?docs=true)
- - [Ledger Nano X firmware v2.0.1](https://support.ledger.com/hc/en-us/articles/360014980580-Ledger-Nano-X-firmware-release-notes?docs=true)
-
-另外，您需要MetaMask作为Ledger设备和Moonbeam网络之间的媒介。确保您的MetaMask[已连接至Moonbeam](/tokens/connect/metamask/)。
-
-自[MetaMask 10.5.0版本](https://consensys.net/blog/metamask/metamask-and-ledger-integration-fixed/)起，在Google Chrome上将您的Ledger设备连接至MetaMask颇为方便，您只需安装最新版本的MetaMask即可。
-
+ - [Ledger Live 2.35.1](https://support.ledger.com/hc/en-us/articles/360020773319-What-s-new-in-Ledger-Live-?docs=true){target=_blank}
```
[the remark this answers is not in the document]
Comment (themacexpert) [reply .snippets/text/ledger/checking-prereqs.md:7]: Ledger has docs in Chinese, perhaps we could use those? Github formats the link a bit strangely by replacing the Chinese characters but it works fine.

**Review by themacexpert — APPROVED** [review]
Looks good! Sorry for taking a while to review this.

File: .snippets/text/ledger/checking-prereqs-ll.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+自2022年11月29日起，Moonbeam和Ledger Live已完成集成，允许用户直接在Ledger Live使用Ledger设备发送和接收GLMR Token。通过此次集成，您无需将Ledger连接至MetaMask。如果您想通过此方法操作，请直接跳转至本教程的[使用Ledger Live发送&接收GLMR](#use-ledger-live)部分。
+
+如果您更偏向使用MetaMask作为媒介来完成Ledger设备与Moonbeam之间的操作，请确保您的MetaMask[已连接至Moonbeam](/tokens/connect/metamask/){target=_blank}。
+
+自[MetaMask的10.5.0版本](https://consensys.net/blog/metamask/metamask-and-ledger-integration-fixed/){target=_blank}发布后，将Ledger设备连接至Chrome的操作变得相对简单，您只需要安装最新版本的MetaMask即可。
```
Comment (wuxinxin2000) [inline .snippets/text/ledger/checking-prereqs-ll.md:5]: 自[MetaMask的10.5.0版本](https://consensys.net/blog/metamask/metamask-and-ledger-integration-fixed/){target=_blank}发布后，将Ledger设备连接至Chrome上的MetaMask操作变得相对简单，您只需要安装最新版本的MetaMask即可。

File: .snippets/text/ledger/ledger-live.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+1. 从左侧菜单栏点击**Receive**
+2. 随后会跳出弹窗，您可以在**Account to credit**下拉菜单中选择您想要用于接收Token的Moonbeam账号
+3. 点击**Continue**
+
+![Verify receiving address in Ledger Live](/images/tokens/connect/ledger/moonbeam/ledger-13.png)
+
+接下来，您的地址应该会出现在Ledger Live上，并提示您在Ledger上验证地址。在您的设备上执行以下步骤：
+
+1. 您应该在您的设备屏幕上看到**Verify Address**。点击右侧按钮开始验证地址
+2. 在下一个显示屏幕中，您应该会看到您的地址。将您设备上的地址与Ledger Live上显示的地址进行比较，并验证其是否匹配。同时，您需要从Ledger Live复制地址用于发送交易。点击按钮继续下一步
```
Comment (wuxinxin2000) [inline .snippets/text/ledger/ledger-live.md:35]:
```suggestion
2. 在下一个显示屏幕中，您应该会看到您的地址。将您设备上的地址与Ledger Live上显示的地址进行比较，并验证其是否匹配。同时，您需要从Ledger Live复制地址用于发送交易。点击右侧按钮继续下一步
```

File: .snippets/text/ledger/checking-prereqs-ll.md (hunk 5 above)
Comment (wuxinxin2000) [inline .snippets/text/ledger/checking-prereqs-ll.md:5]:
```suggestion
自[MetaMask的10.5.0版本](https://consensys.net/blog/metamask/metamask-and-ledger-integration-fixed/){target=_blank}发布后，将Ledger设备连接至Chrome上的MetaMask操作变得相对简单，您只需要安装最新版本的MetaMask即可。
```

File: builders/build/eth-api/dev-env/tenderly.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+- **[Debugger](https://docs.tenderly.co/debugger/how-to-use-tenderly-debugger){target=_blank}** - 使用可视化调试器检查交易并更好地了解代码。通过debugger，您可以查看交易的堆栈追踪，查看交易中的调用，识别合约并查看编码输入、输出和状态变量。您可以在Tenderly数据面板或[Tenderly Debugger谷歌扩展程序](https://docs.tenderly.co/simulations-and-forks/how-to-simulate-a-transaction){target=_blank}使用debugger
+
+- **[Gas Profiler](https://docs.tenderly.co/debugger/how-to-use-tenderly-debugger#gas-profiler){target=_blank}** - 查看您更细化的gas花费情况，从而您可以优化您的智能合约并减少交易gas成本
+
+- **[Simulator](https://docs.tenderly.co/simulations-and-forks/how-to-simulate-a-transaction){target=_blank}** - 在分叉的开发环境中模拟交易，以在无需将交易传送至链上的情况下了解其如何运作。通过这种方式，您可以了解交易的结果并确保在将其发送至网络前按预期运作。您可以尝试不同的参数，模拟历史和当前交易，并编辑合约源代码。您可以从Tenderly数据面板，或在[Tenderly Simulation API](https://docs.tenderly.co/simulations-and-forks/simulation-api){target=_blank}以编程的方式利用模拟器来访问模拟器
```
Comment (wuxinxin2000) [inline builders/build/eth-api/dev-env/tenderly.md:22]:
```suggestion
- **[Simulator](https://docs.tenderly.co/simulations-and-forks/how-to-simulate-a-transaction){target=_blank}** - 在分叉的开发环境中模拟交易，以在无需将交易传送至链上的情况下了解其如何运作。通过这种方式，您可以了解交易的结果并确保在将其发送至网络前按预期运作。您可以尝试不同的参数，模拟历史和当前交易，并编辑合约源代码。您可以从Tenderly数据面板访问模拟器，或用[Tenderly Simulation API](https://docs.tenderly.co/simulations-and-forks/simulation-api){target=_blank}以编程的方式来利用模拟器
```

File: builders/build/eth-api/dev-env/tenderly.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+
+- **[Forks](https://docs.tenderly.co/simulations-and-forks/how-to-create-a-fork){target=_blank}** - 此功能在隔离环境中模拟实时Moonbeam网络，这使您可以与部署的合约和实时链上数据进行交互。您能够按时间顺序将多个模拟链接在一起，分叉还将使交易模拟更进一步。Forks允许复杂的交易测试场景，当出现一个交易依赖于另一个交易，并具有使用实时链上数据的好处。使用Tenderly分叉功能的时候需要注意一些限制。您不能与任何Moonbeam预编译的合约及其功能交互。预编译是Substrate实施的一部分，因此不能在模拟的EVM环境中复制。这禁止您在Moonbeam和基于Substrate的功能（如质押和治理）上与跨链资产交互
```
Comment (wuxinxin2000) [inline builders/build/eth-api/dev-env/tenderly.md:24]:
```suggestion
- **[Forks](https://docs.tenderly.co/simulations-and-forks/how-to-create-a-fork){target=_blank}** - 此功能在隔离环境中模拟实时Moonbeam网络，这使您可以与部署的合约和实时链上数据进行交互。分叉还使交易模拟更进了一步，它使您能够按时间顺序将多个模拟链接在一起。Forks允许复杂的交易测试场景，当出现一个交易依赖于另一个交易，并具有使用实时链上数据的好处。使用Tenderly分叉功能的时候需要注意一些限制。您不能与任何Moonbeam预编译的合约及其函数交互。预编译是Substrate实施的一部分，因此不能在模拟的EVM环境中复制。这禁止您在Moonbeam上与跨链资产交互和使用基于Substrate的功能（如质押和治理）
```

File: builders/build/eth-api/dev-env/tenderly.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+- **[Alerting](https://docs.tenderly.co/alerts/alerting){target=_blank}** - 配置实时警报，在特定活动发生时通知，以便让您了解智能合约的情况
+
+- **[Web3 Actions](https://docs.tenderly.co/web3-actions/intro-to-web3-actions){target=_blank}** - 在JavaScript或TypeScript创建可编程函数，当特定智能合约或链事件发生时，此函数能够通过Tenderly自动执行
+
+- **[Analytics](https://docs.tenderly.co/analytics/general-analytics){target=_blank}** - 可视化交易和链上数据以深入了解当前项目进展。您可以使用Tenderly的分析构建器或创建自定义查询和脚本来满足您的分析需求
+
+- **[Sandbox](https://sandbox.tenderly.co/){target=_blank}** - 在您的浏览器使用内置的JavaScript和Solidity编辑器来直接编写、编译、执行和调试您的智能合约。每次您运行您的代码，Tenderly会创建一个临时分叉，其中包含10个预注资的账户，每个账户有用于测试使用的100个Token
+
+!!! 注意事项
+    对Moonbeam和Moonriver的支持现分为两个阶段。目前处于第一阶段的集成，您可以查看[Tenderly文档网站的限制内容](https://docs.tenderly.co/supported-networks-and-languages#footnotes){target=_blank}。在第二阶段，对Moonbeam和Moonriver的支持将完全启动。
```
Comment (wuxinxin2000) [inline builders/build/eth-api/dev-env/tenderly.md:35]:
```suggestion
    对Moonbeam和Moonriver的支持现分为两个阶段实现。目前集成正处于第一阶段，您可以[在Tenderly文档网站上查看局限性](https://docs.tenderly.co/supported-networks-and-languages#footnotes){target=_blank}。在第二阶段，对Moonbeam和Moonriver的支持将完全启动。
```

File: builders/build/eth-api/dev-env/tenderly.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+
+以下部分将向您展示如何开始在Moonbeam上使用Tenderly。更详细的文档，请参考[Tenderly'的文档网站](https://docs.tenderly.co/){target=_blank}
+
+### 创建Sandbox {: #create-a-moonbeam-sandbox }
+
+要使用Tenderly Sandbox部署合约至Moonbeam，您可以导向至[sandbox.tenderly.co](https://sandbox.tenderly.co/){target=_blank}并执行以下步骤：
+
+1. 在左手边的Solidity编辑器中输入您的智能合约
+2. 从**Network**菜单中选择**Moonbeam**或**Moonriver**，调整任何编译设置，并在需要时指定要在之上运行代码的区块
+3. 更新合约右侧的JavaScript编辑器。Ethers.js和Web3.js默认包含在Sandbox中，并可分别用`ethers`和`web3`实例化。另外需要注意的是Sandbox包含[global variables](https://docs.tenderly.co/tenderly-sandbox#available-javascript-global-variables){target=_blank}以简化开发，因此您无需更新Moonbeam的RPC URL
```
Comment (wuxinxin2000) [inline builders/build/eth-api/dev-env/tenderly.md:55]:
```suggestion
3. 为合约更新右侧的JavaScript编辑器。Ethers.js和Web3.js默认包含在Sandbox中，并可分别用`ethers`和`web3`实例化。另外需要注意的是Sandbox包含[global variables](https://docs.tenderly.co/tenderly-sandbox#available-javascript-global-variables){target=_blank}以简化开发，因此您无需更新Moonbeam的RPC URL
```

File: builders/build/eth-api/dev-env/tenderly.md [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+
+![Contract in list of contracts](/images/builders/build/eth-api/dev-env/tenderly/tenderly-4.png)
+
+要充分利用Tenderly工具集，建议您验证您的智能合约，您可通过点击**Verify**按钮完成此操作。您可以通过上传合约的JSON、ABI或源代码验证您的合约。更多信息，请参考[Tenderly的文档](https://docs.tenderly.co/monitoring/smart-contract-verification/verifying-a-smart-contract#verifying-a-smart-contract){target=_blank}
+
+### 创建分叉 {: #fork-moonbeam }
+
+Tenderly的分叉功能在隔离环境中模拟实时Moonbeam网络，这使您能够与部署的合约和实时链上数据进行交互。
+
+使用Tenderly分叉功能的时候需要注意一些限制。您不能与任何Moonbeam预编译的合约及其功能交互。预编译是Substrate实施的一部分，因此不能在模拟的EVM环境中复制。这禁止您在Moonbeam和基于Substrate的功能（如质押和治理）上与跨链资产交互。
```
Comment (wuxinxin2000) [inline builders/build/eth-api/dev-env/tenderly.md:86]:
```suggestion
使用Tenderly分叉功能的时候需要注意一些限制。您不能与任何Moonbeam预编译的合约及其功能交互。预编译是Substrate实施的一部分，因此不能在模拟的EVM环境中复制。这禁止您在Moonbeam上与跨链资产交互和使用基于Substrate的功能（如质押和治理）。
```

File: .snippets/text/ledger/checking-prereqs.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (eshaben) [reply .snippets/text/ledger/checking-prereqs.md:7]: Good idea!

**Comment by eshaben** [comment]
Thanks for the feedback @themacexpert and @wuxinxin2000 !!

**Pull request merged**